scheduler/src/cook/kubernetes/compute_cluster.clj

```diff
@@ -242,6 +243,55 @@
 
   (restore-offers [this pool-name offers])
 
+  (autoscaling? [_]
+    (and
+      (some-> synthetic-pods-config :image count pos?)
+      (some-> synthetic-pods-config :user count pos?)
+      (some-> synthetic-pods-config :max-pods-outstanding pos?)))
+
+  (autoscale! [this pool-name task-requests]
```

Comment:
As a bigger picture comment, we do not want to name these pods with with their task-id, then we'll find mysterious task-id's in the log. That's goint to be confusing diagnostics. Much better to indicate it in the pod name directly.

E.g., instead of synthesizing labels here, send a 'synthetic pod' flag to task-metadata->pod. Have it syntehsize the labels and have it explicitly name a pod with "syn-UUID", that way we can identify when we see them enter the state machine because of their unique name. That way if I see the name, I know where it comes from.... and not to find it in the database. 


database. My suggestion is when we're

Reply:
Good idea, will do

Reply:
We're now using `synthetic-{pool}-{d/squuid}`